bgpd/bgp_bmp.c

```diff
@@ -1773,8 +1776,38 @@ static void bmp_active_connect(struct bmp_active *ba)
 {
 	enum connect_result res;
 	char buf[SU_ADDRSTRLEN];
+	struct interface *ifp;
+	vrf_id_t vrf_id;
```

Comment:
I suggest initializing this to VRF_DEFAULT.

Reply:
taken into account